test/javascript/explore-data/search-matrix-spec.js

```diff
@@ -1,12 +1,17 @@
 describe("Searching a matrix (table) of data...", function() {
-
+	var tooSlowSeconds = 30;
 	var debug = function() {
+		return;
 		console.log(arguments);
```

Comment:
this turned off the debug function so that the console logs werent running (they slow down the script)